runtime/lua/vscode/cursor.lua

```diff
@@ -1,4 +1,5 @@
 local util = require("vscode.util")
+local vscode = require("vscode")
```

Comment:
This crashes because it requires vscode which requires cursor which requires vscode. What is the correct solution to this?

Reply:
Why does vscode require cursor? 

If the vscode.lua module is intended to be the thing that "gathers" all the necessary modules, then don't put any actual logic in vscode.lua.